tests/test_properties.py

```diff
@@ -0,0 +1,14 @@
+import brickschema
+from brickschema.namespaces import BRICK, RDFS
+import sys
+
+sys.path.append("..")
+from bricksrc.properties import properties  # noqa:E402
```

Comment:
Maybe we can change the file name and the object name to relationships from now on to be explicit on the naming? It's not critical but would be nice to have.

Reply:
Nice idea! I'll see how easy it is to make that adjustment